packages/graphql/src/services/participants.ts

```diff
@@ -1,8 +1,4 @@
-import {
-  MicroSessionStatus,
-  PublicationStatus,
-  SessionStatus,
-} from '@klicker-uzh/prisma'
+import { PublicationStatus, SessionStatus } from '@klicker-uzh/prisma'
```

Comment:
Publication status is also the wrong one here to filter for micrlearning content